Sources/Commands/PackageTools/Init.swift

```diff
@@ -83,4 +83,8 @@ extension SwiftPackageTool {
     }
 }
 
+#if swift(<5.10)
```

Comment:
If these are for `main` only, this should be 5.11, right?

Reply:
Good point, I thought the attribute is available in 5.10 already, but the warning itself was introduced in 5.11. But actually this fails to build with 5.10.